src/components/views/ClipBox.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+
+import './css/ClipBox.css';
+
+
+class ClipBox extends Component {
+    constructor(props) {
+        super(props);
+
+        this.inputRef = React.createRef();
+    }
+
+    handleInputFocus = e => {
+        if (e.target) e.target.select();
+    }
+
+    handleCopyClick = e => {
+        if (!e.target) return;
+        e.target.parentNode.querySelector('input').select();
+        document.execCommand('copy');
+    }
```

Comment:
This doesn't need a container or access to its own state. Just does simple DOM interaction. I could wrap it in a useless container or extract out to the experiment container but it would be less reusable esp if we want it in other projects.